wetmap/src/components/newModals/setting/index.tsx

```diff
@@ -0,0 +1,79 @@
+
+import SettingsView from './view';
+
+import { SessionContext } from '../../contexts/sessionContext';
+import { useContext, useState, useEffect } from 'react';
+import React from 'react';
+
+
+export default function Settings(props) {
+  const { activeSession, setActiveSession } = useContext(SessionContext);
```

Comment:
this one is complaining because activeSession, setActiveSession are not being used, you can either remove it or comment it out if you are planning to use it later

Reply:
I'm still seeing these being imported, why did you mark this as resolved with out making changes or at least documenting your reasoning in here as a response?